openpower/platform/Config.in

```diff
@@ -2,3 +2,11 @@ config BR2_OPENPOWER_PLATFORM
     bool "Enable the OpenPower Platform"
     default n
     depends on BR2_powerpc64 || BR2_powerpc64le
+
+config BR2_OPENPOWER_POWER8
```

Comment:
Need to at least make these depend on the ! Of the other one so only one can be set.